atlp-rwanda/script-heroes-frontend
# #174797498 View/Post messages on the Chat system

**What does this PR do?**
View and Post messages on the chat

**Description of Task to be completed?**
- Create Route to the Chat page (Link in the Navigation system)
- Create component displaying posted messages
- Save posted messages in DB and display them (in Realtime)

**How should this be manually tested?**
- Clone this repo git clone `https://github.com/atlp-rwanda/script-heroes-frontend`
- Checkout the branch git checkout `ft-chat-system-#174797498`
- Install dependencies yarn install
- Run app yarn start
- Log in (use 2 browsers to perceive real-time functionality):
              - 1st browser: email: will@email.com,
              - 2nd browser: email: john@email.com
              - Both side, use `Password123` as password
- Then navigate to the Chat page
- The user should be able to view the messages posted in Realtime
- The user should be able to post messages

**What are the relevant pivotal tracker stories?**
[#174797498](https://www.pivotaltracker.com/story/show/174797498)
